core/management/generate_data.py

```diff
@@ -0,0 +1,83 @@
+from django.core.management import BaseCommand
+
+from spaces.models import Space
+from posts.models import Post
+from users.models import User
+from users.tests.factories import UserFactory
+
+class Command(BaseCommand):
+    help = "Creates a random local data for testing."
+
+    def handle(self, *args, **kwargs):
+        for _ in range(10):
+            UserFactory()
+
+        spaces = [
+            Space(name="Cat Memes"),
+            Space(name="Tech Geeks"),
+            Space(name="Pizza Lovers"),
+            Space(name="Coffee Addicts"),
+        ]
+        Space.objects.bulk_create(spaces)
+
+        print("Mock data created.")
+
+for space in spaces:
```

Comment:
These should be inside of the handle function. Please correct this and push it again.
If you want to learn more about what we are doing here:
- https://simpleisbetterthancomplex.com/tutorial/2018/08/27/how-to-create-custom-django-management-commands.html
- https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/